### DISCUSSION TITLE
biome-ignore entire file

### DISCUSSION THREAD

**Discussion by MakakWasTaken** [biomejs/biome#605] [Q&A] [answered]
Is it possible to ignore an entire file in biome using a ignore command like: 
```
// biome-ignore ...
```
In the same way that:
```
/* eslint-disable ... */
```

When attemping to use in the same way that eslint provides I get the following: `Suppression comment is not being used`. So I assume that it is not implemented in biomejs yet?

**Comment by ematipico** [comment, marked as answer]
This is being worked as part of Biome V2

https://github.com/biomejs/biome/issues/4305

**Reply by tobimori** [reply]
Example of how to do this in v2, since this still ranks on first place on Google:
```// biome-ignore-all lint/suspicious/noDebugger: reason```

**Reply by enBonnet** [reply]
`// biome-ignore-all lint: reason`
If you want to disable it all

**Reply by dimasivan** [reply]
`// biome-ignore-all lint: reason` as the first line of the file
